schemas/compat-data-schema.md

```diff
@@ -198,8 +198,8 @@ information about versions, prefixes, or alternate names, as well as notes.
 The currently accepted browser identifiers should be declared in alphabetical order:
 * `chrome`, Google Chrome (on desktops)
 * `chrome_android`, Google Chrome (on Android)
-* `edge`, MS Edge (on Windows)
-* `edge_mobile`, MS Edge, the mobile version
+* `edge`, MS Edge (on Windows), based on the EdgeHTML version
+* `edge_mobile`, MS Edge, the mobile version, based on the EdgeHTML version
```

Comment:
While we're here, what do you think of specifying that this is the Windows Mobile operating system?

```suggestion
* `edge_mobile`, MS Edge, the Windows Mobile version, based on the EdgeHTML version
```

Reply:
Hands down, yes.